content/en/dashboards/template_variables.md

```diff
@@ -76,6 +76,11 @@ After the widget is saved, the value of the template variable is the one selecte
 
 {{< img src="dashboards/template_variables/selecting_template_variables.png" alt="Selecting template variables"  style="width:75%;">}}
 
+#### Associated template variables
+When selecting a template variable value, the **Associated Values** and **Other Values** sections are displayed. Associated values are calculated by taking into  the other template variable values selected on the page, and seamlessly identifies the related values without any configuration. 
```

Comment:
@MirandaKapin Hi, it still looks like there's a word missing in the second sentence? I can't guess at what it is from the context given. Should it be "consideration"?